src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
So convianiance, so why it is convenient to write four files include them in big argument list of the function call and use alias instead of just doing `window.setTimeout`?